allennlp/data/dataset_readers/wikitables/wikitables_dataset_reader.py

```diff
@@ -0,0 +1,141 @@
+        utterance_field = TextField(tokenized_utterance, self._utterance_token_indexers)
+        table_filename = "%s/%s" % (self._tables_directory, parsed_info["context"])
+        # We want the TSV file, but the ``*.examples`` files typically point to CSV.
+        table_filename = table_filename.replace(".csv", ".tsv")
+        table_knowledge_graph = TableKnowledgeGraph.read_from_file(table_filename)
+        table_field = KnowledgeGraphField(table_knowledge_graph, self._utterance_token_indexers)
+        if for_train:
+            world = World(table_knowledge_graph)
+            dpd_output_filename = "%s/%s.gz" % (self._dpd_output_directory, parsed_info["id"])
+            sempre_forms = [line.strip().decode('utf-8') for line in gzip.open(dpd_output_filename)]
+            expressions = world.process_sempre_forms(sempre_forms)
+            action_sequences = [world.get_action_sequence(expression) for expression in expressions]
+            action_sequences_field = ListField([self._make_action_sequence_field(sequence)
+                                                for sequence in action_sequences])
+            return Instance({"utterance": utterance_field,
+                             "context": table_field,
+                             "action_sequences": action_sequences_field})
+        else:
+            return Instance({"utterance": utterance_field, "context": table_field})
+
+    @staticmethod
+    def _make_action_sequence_field(action_sequence: List[str]) -> ListField:
+        action_sequence.insert(0, START_SYMBOL)
+        action_sequence.append(END_SYMBOL)
+        return ListField([LabelField(action, label_namespace='actions') for action in action_sequence])
+
+    @staticmethod
+    def _parse_line_as_lisp(lisp_string: str) -> Dict[str, LispValueType]:
+        parsed_info = {}
+        input_nested_list = pyparsing.OneOrMore(pyparsing.nestedExpr()).parseString(lisp_string).asList()[0]
+        for key_value in input_nested_list:
+            if not isinstance(key_value, list):
+                continue
+            if key_value[0] == "id":
+                parsed_info["id"] = key_value[1]
+            elif key_value[0] == "utterance":
+                parsed_info["utterance"] = key_value[1].replace("\"", "")
+            elif key_value[0] == "context":
+                parsed_info["context"] = key_value[1][-1]
+            elif key_value[0] == "targetValue":
+                parsed_info["targetValue"] = [x[1] for x in key_value[1][1:]]
+        assert all([x in parsed_info for x in ["id", "utterance", "context"]]), "Invalid format"
+        return parsed_info
+
+    @classmethod
+    def from_params(cls, params: Params) -> 'WikitablesDatasetReader':
+        tables_directory = params.pop('tables_directory')
+        dpd_output_directory = params.pop('dpd_output_directory', None)
+        utterance_tokenizer_type = params.pop('utterance_tokenizer', None)
```

Comment:
You can just do this: https://github.com/allenai/allennlp/blob/c4d80d0b68d575e831cef2e249b844db15b5e228/allennlp/data/dataset_readers/reading_comprehension/triviaqa.py#L166-L167

Reply:
Done.